### ISSUE TITLE
Option to use last amount value from the same payee (autocompletion).

### ISSUE THREAD

**Issue by medismail** [moneymanagerex/android-money-manager-ex#1744]
**Is your feature request related to a problem? Please describe.**
Each time I add a new transaction I should fill again the amount and notes, but in many cases this values are the same with same payee.

**Describe the solution you'd like**
If it possible to add an option (like the option to use last payee for new transaction) but this option just fill the amount, the category and notes with the last transaction values in these fields after select the payee in the new transaction

**Describe alternatives you've considered**
No alternative option.

**Additional context**
To summarize: 
If the new option is enabled.
 - Create new transaction
 - choose the payee
 - the application fill the amount, the category and the note fields with values of the same fields in the last transaction with the same payee

 I was use desktop application before name is grisbi that's have the feature and really so useful, and I notice also that it was suggested before in this #1367 .
This feature is similar to recurring transaction but without a periodic time, I know can copy transaction but it is mean each time make search for transaction.

**Comment by wolfsolver** [comment]
Actually chosen a payee will fill category with last used for that payee 

Desktop version has the same behaviour.

I think that both app need to have the same behaviour. Why to you think on mobile need to be different?

**Comment by medismail** [comment]
Thank you for reply, I think it can be also a new feature for desktop application. As I use actually the mobile app I was open the request here.

**Comment by guanlisheng** [comment]
hi @medismail, this `Default Payee`  (#1634) might be what you want. please have a check.


`Settings -> General -> Default Payee 
`
![Screenshot_20240327_155939](https://github.com/moneymanagerex/android-money-manager-ex/assets/721973/db20c1b5-eb5e-4bda-ab95-eba4cecd0f4d)

**Comment by medismail** [comment]
The default payee, when create a new transaction add the last payee
What I am looking for is to fill the amount field with value of last value from same payee for example each time I am going to doctor I pay 43 dollars, after created a new transaction and choose payee doctor the amount should be fill with 43.
This little similar with repeated transaction but is not have a periodic time.

**Comment by guanlisheng** [comment]
I c, this is a nice-to-have feature.

**Comment by luisdev** [comment]
This would be a nice feature. It's like _Scheduled Bills & Deposits_ in Quicken (Reckon).
